src/Microsoft.AspNetCore.Mvc.Core/RequireHttpsAttribute.cs

```diff
@@ -43,6 +47,13 @@ protected virtual void HandleNonHttpsRequest(AuthorizationFilterContext filterCo
                     request.Path.ToUriComponent(),
                     request.QueryString.ToUriComponent());
```

Comment:
I integrated the new feature from `HostString`, can you confirm it's ok from your side?